.github/workflows/hugo.yml

```diff
@@ -29,20 +29,43 @@ jobs:
   # Build job
   build:
     runs-on: ubuntu-latest
-    env:
-      HUGO_VERSION: 0.99.0
     steps:
-      - name: Install Hugo CLI
-        run: |
-          wget -O ${{ runner.temp }}/hugo.deb https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_extended_${HUGO_VERSION}_Linux-64bit.deb \
-          && sudo dpkg -i ${{ runner.temp }}/hugo.deb
-      - name: Checkout
+      - name: checkout repository
         uses: actions/checkout@v3
+
+      - name: Get Hugo version
+        run: echo "HUGO_VERSION=$(cat .hugo_version)" >> $GITHUB_ENV
```

Comment:
Is this our innovation? `.hugo_version`

Reply:
yes, was creating it in based on the python version file we are using in some projects — trying to remove redundancy, previously we were specifying hugo version as an ENV in multiple different workflow files with a note to update them all when we change versions.

My other idea was to stick it somewhere in `package.json` but wasn't sure if you can put arbitrary info there.

Reply:
I like it!